testapp/src/main/res/values/strings.xml

```diff
@@ -6,6 +6,7 @@
     <string name="lb_app_settings">Settings</string>
     <string name="lb_preview_mode">Preview Mode</string>
     <string name="lb_require_signature_verification">Require Signature Verification</string>
+    <string name="lb_enable_prod_version">PRD</string>
```

Comment:
We can elaborate it to `PROD`, `Prod` or, `Production mode` 🤔

Reply:
@khairul-alam-licon It was written in the ticket description
`The text of the toggle should say 'PRD'`